### PR TITLE
Recipe for Perl File::Which module

### PR DESCRIPTION
**Pull request by pvanheus**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks Peter! This one needs a rebase.

File: recipes/perl-forks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  name: perl-forks
+  version: "0.36"
+
+source:
+  fn: forks-0.36.tar.gz
+  url: http://search.cpan.org/CPAN/authors/id/R/RY/RYBSKEJ/forks-0.36.tar.gz
+  md5: 48a1819dd2ff67d756f268771e1a9827
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/perl-forks/meta.yaml]: This can be zero its a new recipe isn't it?

**Comment by bgruening** [comment]
@pvanheus can you reabase and change the build number?

**Comment by bgruening** [comment]
@pvanheus are you working in this?

**Comment by pvanheus** [comment]
Thanks for the ping, just started working on it again @bgruening

**Comment by pvanheus** [comment]
Closing this PR - a newer version of File::Which has already been added to bioconda.

**Pull request closed without merging**